validator/client/validation_client.go

```diff
@@ -144,14 +147,20 @@ func (c *ValidationClient) Room() int {
 }
 
 type ExecutionClient struct {
```

Comment:
normal/rpc-based "ExecutionClient" should not really be affected by this change.